VideoAPI/Video.API/Video.API.csproj

```diff
@@ -13,6 +13,10 @@
     </DocumentationFile>
     <UserSecretsId>9AECE566-336D-4D16-88FA-7A76C27321CD</UserSecretsId>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <UseAppHost>false</UseAppHost>
```

Comment:
whats this

Reply:
There's a bug with dotnet core  certs and catalina